FTPizza.Tests/FTP.Test.cs

```diff
@@ -1,11 +1,28 @@
 ﻿using NUnit.Framework;
 using System.Collections.Generic;
+using System.IO;
+using System;
 
 namespace FTPizza.Tests
 {
     [TestFixture]
     public class FTPTests
     {
+        public string[] connectionFile = File.ReadAllLines(@"connection.txt");
+        public string ftpUrl;
+        public string ftpUser;
+        public string ftpPassword;
+        public Ftp client;
+
+        [SetUp]
+        public void Init()
+        {
+            ftpUrl = connectionFile[0].Replace("url=", "");
+            ftpUser = connectionFile[1].Replace("user=", "");
+            ftpPassword = connectionFile[2].Replace("pass=", "");
+            client = new Ftp(ftpUser, ftpPassword, ftpUrl);
+        }
+
```

Comment:
I really dislike that this unit test has a dependency by interacting with the connectionFile